benchmarks/benchmarks.py

```diff
@@ -132,13 +128,7 @@ def setup(self, tarfile_path):
         tempdir = osp.dirname(tarfile_path)
         with tarfile.open(tarfile_path) as tar:
             tar.extractall(tempdir)
-
-        # TODO -- remove this abomination after https://github.com/datalad/datalad/issues/1512 is fixed
-        epath = opj(tempdir, 'testds1')
-        epath_unique = epath + str(SuperdatasetsOperationsSuite.ds_count)
-        os.rename(epath, epath_unique)
-        SuperdatasetsOperationsSuite.ds_count += 1
-        self.ds = Dataset(epath_unique)
+        self.ds = Dataset(opj(tempdir, 'testds1'))
```

Comment:
actually I will have to revert this change since we still might need to benchmark old commits thus should be ready to workaround.